### DISCUSSION TITLE
Releases from Trakt collection

### DISCUSSION THREAD

**Discussion by joshali1990** [itsToggle/plex_debrid#89] [General]
How long after release are episodes in the trakt collection checked? For example the show Industry has a release date of August 23, 2022 2:00 AM, but hasn't been scraped yet. Could it be because the latest episode doesn't get collected and the show isn't in my trakt watchlist?

**Comment by itsToggle** [comment]
once the UTC air-date and time have passed for an episode, plex_debrid should try to download it during the regular 30min interval. 

Have you added your collection to the monitored trakt lists (/Settings/Content Services/Trakt/Trakt lists/)?

Also make sure that the episode isn't marked as collected.

If you have checked both those things and all is in order, I will need to have a look at my code again.

**Reply by joshali1990** [reply]
Yes my trakt collection is monitored. The show is collected in trakt except for the lastest episode. I use plex as my library service.

**Reply by itsToggle** [reply]
Hm, I really don't see anything wrong with the code. Has monitoring the collection to download newly released episodes before? Perhaps this is a one-off error and the trakt api is not updating the metadata correctly. I'll experiment a bit though.
I did notice that "House of the dragon" wasn't added to my server until last night (so a day late).

**Reply by itsToggle** [reply]
Could you check your ignored media (from the main menu) if the episode has been ignored? I've recreated your circumstances and plex_debrid found the show in my trakt collection and added the episode from last night to my realdebrid.

![Screenshot_20220823-211049_Termux~2](https://user-images.githubusercontent.com/71379623/186245167-918d5b7c-3f06-4167-9daa-00cde937cd8c.jpg)

This was after a fresh restart though. Could you restart the script and check if that changes anything?

**Reply by joshali1990** [reply]
Nope not in my ignore list. I wonder if there is some sort of API limit? I have 140 shows in my collection and 70 in my watchlist. 

![image](https://user-images.githubusercontent.com/87837562/186247903-637b487f-47bd-4641-beaa-a9db3c87c478.png)

**Reply by itsToggle** [reply]
You are allowed 1000 calls / 5 min. 2 calls are made to get all watchlisted and collected content. For each show that has less collected episodes than aired episodes, 1 more call is made to get necessary info on seasons. So I dont think you hit any limit :thinking: 

Do you also have this show in your Plex watchlist? Edit: even if that were so, downloading still works in my testing.

**Reply by itsToggle** [reply]
You could turn on debug printing and look for any errors that plex_debrid might print.

**Reply by joshali1990** [reply]
Yep my trakt list auto syncs to my Plex watchlist. I removed it from my Plex watchlist and it scraped for it. I guess the release date without a time from Plex was taking precedence over trakt?

**Reply by itsToggle** [reply]
Once you connect plex_debrid to trakt (regardless of wether or not you actually monitor any trakt lists), all release times will be determined by trakt. I dont know why having a show both in a trakt and a plex watchlist caused this behaviour for you (since I tested this exact situation aswell), but I will keep an eye on it. Tomorrow a new episode from one of the shows I'm monitoring is released, I'll see what happens :)

**Reply by itsToggle** [reply]
Okay, so the newly released episode was scraped for and downloaded like expected. Please let me know if you encounter this issue again, as I cant seem to reproduce it :)
